weave/trace_server/migrations/002_add_deleted_at.down.sql

```diff
@@ -0,0 +1,67 @@
+ALTER TABLE object_versions DROP COLUMN deleted_at;
```

Comment:
This is a hard file to parse given that it is mostly a copy. Can you please comment to make it clear what changed vs what is just the copy over?